src/fmu/ensemble/realization.py

```diff
@@ -1016,20 +1016,21 @@ def load_smry(
 
         Args:
             time_index: string indicating a resampling frequency,
-               'yearly', 'monthly', 'daily', 'last' or 'raw', the latter will
-               return the simulated report steps (also default).
+               'yearly', 'monthly', 'daily', 'first', 'last' or 'raw', the
+               latter will return the simulated report steps (also default).
                If a list of DateTime is supplied, data will be resampled
                to these.
             column_keys: list of column key wildcards. None means everything.
             cache_eclsum: boolean for whether to keep the loaded EclSum
                 object in memory after data has been loaded.
             start_date: str or date with first date to include.
                 Dates prior to this date will be dropped, supplied
-                start_date will always be included.
+                start_date will always be included. Overriden if time_index
```

Comment:
search for this typo globally..

Reply:
copy-paste propagates mistakes fast 😆 will fix 👍